util/cmake/SeqAnBuildSystem.cmake

```diff
@@ -321,7 +322,7 @@ macro (seqan_build_system_init)
         find_package(ZLIB)
         find_package(BZip2)
         find_package(Boost)
-        find_package(SeqAn REQUIRED)
+        find_package(SeqAn CONFIG REQUIRED)
```

Comment:
Why is this required? And is it also required for the apps when built individually, i.e. did you try `SEQAN_BUILD_SYSTEM=SEQAN_RELEASE_APPS` ?